easybuild/framework/easyblock.py

```diff
@@ -2982,6 +2978,39 @@ def build_and_install_one(ecdict, init_env):
     return (success, application_log, errormsg)
 
 
+def reproduce_build(app, reprod_dir_root):
```

Comment:
@boegel I need some help with how to unit test this, not sure what's required to get a proper `app` instance

Reply:
...or perhaps what is already tested is good enough?

Reply:
@ocaisa Once you have a parsed easyconfig (for example for `toy-0.0.eb` in the tests, you can just create an easyblock instance like `MyEasyBlock(parsed_easyconfig)` (e.g. `Toy(toy_ec)`).

But I guess there's no strong need to test this function in isolation, since it's always being called. Just make sure whatever it produces it being checked, which you're already doing I think.